backends/dnf/meson.build

```diff
@@ -1,6 +1,11 @@
 appstream_dep = dependency('appstream-glib')
 dnf_dep = dependency('libdnf', version: '>=0.43.1')
 rpm_dep = dependency('rpm')
+c_args = ['-DG_LOG_DOMAIN="PackageKit-DNF"']
+
+if meson.get_compiler('c').has_function('hy_query_get_advisory_pkgs', prefix: '#include <libdnf/hy-query.h>', dependencies: dnf_dep)
+   c_args += ['-DHAVE_HY_QUERY_GET_ADVISORY_PKGS=1']
```

Comment:
I don;t think you need the `=1`

Reply:
right, it's not needed, but it also doesn't hurt. I can change it.